kestrel-inmemory-backend/src/main/kotlin/com/dreweaster/ddd/kestrel/infrastructure/InMemoryBackend.kt

```diff
@@ -25,11 +27,107 @@ import java.time.Instant
 import java.util.UUID
 import kotlin.reflect.KClass
 
+interface InMemoryEventStreamHandler {
```

Comment:
These changes were to allow the in-memory-stream handle event serialisation for streaming